### PR TITLE
fix: #2520 - Refactor app list + analytics page styles

### PR DESCRIPTION
**Pull request by nphivu414**
# Pull request checklist

## Does this close any currently open issues?

fixes: #2520 

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #2520 

## What is the new behavior?

- Refactor App List + Analytics page styles

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by tanphamhaiduong — APPROVED** [review]
LGTM

File: packages/developer-portal/src/components/pages/app-detail/app-authentication-detail/__styles__/app-authentication-detail.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  border: 1px solid ${grey};
+  min-width: 80%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`
+
+export const authenticationCode = css`
+  word-break: break-all;
+  margin-bottom: 0 !important;
```
Comment (tanphamhaiduong) [inline packages/developer-portal/src/components/pages/app-detail/app-authentication-detail/__styles__/app-authentication-detail.ts:15]: Can we remove !important ?

**Review by ant066 — APPROVED** [review]

File: packages/developer-portal/src/components/ui/apps/app-list/app-list.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const AppList: React.FunctionComponent<AppListProps> = ({
             : 'We are unable to find any Apps that match your search criteria. Please try again.'}
         </Helper>
       ) : (
-        <GridFourCol className={` ${loading ? styles.contentIsLoading : ''}`} data-test="app-list-container">
+        <GridFourCol className={` ${loading ? contentIsLoading : ''}`} data-test="app-list-container">
```
Comment (tanphamhaiduong) [inline packages/developer-portal/src/components/ui/apps/app-list/app-list.tsx:31]: Pls use cx

File: packages/developer-portal/src/components/pages/app-detail/app-authentication-detail/__styles__/app-authentication-detail.ts (hunk 1 above)
Comment (nphivu414) [reply packages/developer-portal/src/components/pages/app-detail/app-authentication-detail/__styles__/app-authentication-detail.ts:15]: Hi @tanphamhaiduong ,
I'll fix it in the next PR

**Pull request merged**